elastic/elastic-agent
# Ensure we wait the Elastic-Agent process to finish in TestContainerCMD

## What does this PR do?

Ensure we wait the Elastic-Agent process to finish in TestContainerCMD

## Why is it important?

Closes https://github.com/elastic/elastic-agent/issues/4246

## Checklist


~~## Disruptive User Impact~~

## How to test this PR locally

Run the integration test `TestContainerCMD`, make sure to adjust the test platform.

```
TEST_PLATFORMS="linux/amd64/ubuntu" SNAPSHOT=true mage -v integration:single TestContainerCMD
```

## Related issues

- Closes https://github.com/elastic/elastic-agent/issues/4246


## Questions to ask yourself

- How are we going to support this in production?
- How are we going to measure its adoption?
- How are we going to debug this?
- What are the metrics I should take care of?
- ...
